Server/routes/authRoute.js

```diff
@@ -8,12 +8,14 @@ const {
   recoveryRequestController,
   validateRecoveryTokenController,
   resetPasswordController,
+  deleteUserController
 } = require("../controllers/authController");
 
 //Auth routes
 router.post("/register", registerController);
 router.post("/login", loginController);
 router.post("/user/:userId", verifyJWT, userEditController);
+router.delete("/user/:userId", verifyJWT, deleteUserController);
```

Comment:
We also need to verify ownership, not just authentication.

If we only check authentication a user can use his JWT token to delete any user he wants.

We need to use the verify Ownerships middleware to verify that the user has the right to do that operation.

I modified the middleware today to handle users, so please fetch and merge the latest code to use it.

You can see examples of how the verifyOwnership middleware works in the monitor controller